packages/oid4vci-holder/src/machine/oid4vciMachine.ts

```diff
@@ -271,6 +291,28 @@ const createOID4VCIMachine = (opts?: CreateOID4VCIMachineOpts): OID4VCIStateMach
           },
         },
       },
+      [OID4VCIMachineStates.getFederationTrust]: {
+        id: OID4VCIMachineStates.getFederationTrust,
+        invoke: {
+          src: OID4VCIMachineServices.getFederationTrust,
+          onDone: {
+            target: OID4VCIMachineStates.addContact,
+            actions: assign({
+              isTrustedFederation: (_ctx: OID4VCIMachineContext, _event: DoneInvokeEvent<boolean>) => _event.data
+            })
+          },
+          onError: {
+            target: OID4VCIMachineStates.handleError,
+            actions: assign({
+              error: (_ctx: OID4VCIMachineContext, _event: DoneInvokeEvent<Error>): ErrorDetails => ({
+                title: translate('oid4vci_machine_store_credential_error_title'), // TODO
```

Comment:
TODO what?

Reply:
had to set the correct localization string. fixed